covid19indiatracker_bot.py

```diff
@@ -471,8 +470,8 @@ def main():
         CommandHandler('covid19india', covid19india))
     updater.dispatcher.add_handler(CommandHandler('statecodes', statecodes))
     updater.dispatcher.add_handler(CommandHandler('mohfw', mohfw))
-    updater.dispatcher.add_handler(CommandHandler('mohfwapi', mohfwapi))
-    updater.dispatcher.add_handler(CommandHandler('mohfwsite', mohfwsite))
+    # updater.dispatcher.add_handler(CommandHandler('mohfwapi', mohfwapi))
```

Comment:
I think you can leave the handler to `/mohfwapi` defined and instead remove it from the `/help` section. That way, it doesn't distract the average user and in the event of the web scraper failing(perhaps for a single state or so), devs would still have access to it.

Reply:
Made the changes. https://github.com/cibinjoseph/covid19indiatracker_bot/pull/2/commits/9a642876c35088f4218655eb35ee5fca7899a981